menus/Context.sublime-menu

```diff
@@ -1,7 +1,23 @@
 [
     {
         "id": "sublimelinter",
-        "caption": "Lint This View",
-        "command": "sublime_linter_lint"
+        "caption": "Linter",
+        "children": [
+            {
+                "id": "sublimelinter-lint",
+                "caption": "Lint This View",
+                "command": "sublime_linter_lint"
+            },
+            {
+                "id": "sublimelinter-highlights",
+                "caption": "Toggle Highlights",
+                "command": "sublime_linter_toggle_highlights"
+            },
+            {
+                "id": "sublimelinter-fix",
+                "caption": "Quick Action",
+                "command": "sublime_linter_quick_actions"
+            }
+        ]
     }
```

Comment:
If you know, how users could remove or modify this, maybe add a comment.

Reply:
Removing context menu entries works via ST's standard override protocol. It's not easy, and not well known outside a handful old timers probably, but it's powerful.

- In Packages create a directory with the same name as the package you want to override (in this case "SublimeLinter", but this also works for AlignTab or ColorConvert to name a few).
- Using [PackageResourceViewer](https://packagecontrol.io/packages/PackageResourceViewer) open the Context.sublime-menu file from that package, and save it in the new directory you just created. 
- This file will now replace the one from the installed package. You can override every single file in any package this way.
- To remove the context menu, change the `children` key to an empty array `[]`.

To add entries, create a Context.sublime-menu file in the Packages/User directory (aka the "User package"), create an entry with the `id` that matches the one we ship, and create a child entry you want to add (e.g. `sublime_linter_panel_toggle`).

```json
[
    {
        "id": "sublimelinter",
        "caption": "Linter",
        "children": [
            {
                "id": "sublimelinter-panel",
                "caption": "Show Panel",
                "command": "sublime_linter_panel_toggle"
            }
        ]
    }
]
```

---

I'll add this to our docs.